front_end/content.py

```diff
@@ -1777,6 +1775,12 @@ def move_exercise(self, course_id, assignment_id, exercise_id, new_assignment_id
                           AND assignment_id = ?
                           AND exercise_id = ?''', (new_assignment_id, course_id, assignment_id, exercise_id, ))
 
+        self.execute('''UPDATE submission_outputs
```

Comment:
I forgot but if in move_exercise we're migrating submissions, we need to be migrating submission_outputs as well.